src/components/shared/channel-article/ChannelMoreBtn.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import { useState } from 'react';
+
+import More from '@components/icons/More';
+
+export default function ChannelMoreBtn() {
```

Comment:
별 건 아닌데 해주시고 밑에서 export default ChannelMoreBtn 해도 좋을 것 같아요~!
```suggestion
function ChannelMoreBtn() {
```

Reply:
수정완료!